File: gva/server/router/pmocker/cost_actual.go
package pmocker

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type CostActualRouter struct{}

func (r *CostActualRouter) InitCostActual(public *gin.RouterGroup, private *gin.RouterGroup) {
	{
		costActual := private.Group("costActual").Use(middleware.OperationRecord())
		costActual.POST("create", apiGroup.CostActualApi.CreateCostActual)
		costActual.PUT("update", apiGroup.CostActualApi.UpdateCostActual)
		costActual.DELETE("delete", apiGroup.CostActualApi.DeleteCostActual)
		costActual.POST("confirm", apiGroup.CostActualApi.ConfirmCostActual)
	}
	{
		costActual := private.Group("costActual")
		costActual.GET("find", apiGroup.CostActualApi.FindCostActual)
		costActual.GET("list", apiGroup.CostActualApi.ListCostActuals)
	}
}

File: gva/web/src/api/pmocker/costActual.js
import service from '@/utils/request'

export const createCostActual = (data) => service({ url: '/pmocker/costActual/create', method: 'post', data })
export const updateCostActual = (data) => service({ url: '/pmocker/costActual/update', method: 'put', data })
export const deleteCostActual = (params) => service({ url: '/pmocker/costActual/delete', method: 'delete', params })
export const findCostActual = (params) => service({ url: '/pmocker/costActual/find', method: 'get', params })
export const listCostActuals = (params) => service({ url: '/pmocker/costActual/list', method: 'get', params })
export const confirmCostActual = (params) => service({ url: '/pmocker/costActual/confirm', method: 'post', params })
